worlds/civ_6/Options.py

```diff
@@ -0,0 +1,104 @@
+from dataclasses import dataclass
+from Options import Choice, DeathLink, DefaultOnToggle, PerGameCommonOptions, Range, Toggle
```

Comment:
Would recommend putting Start Inventory From Pool in here too. It's like regular start inventory, but instead of creating a new item to place in the player's starting inventory, it instead takes the item from the item pool and replaces it with a random filler item (generated based off of your `get_filler_item_name` in your world class)

All you need to do to add the option is:
1. In your `from Options` imports, add `StartInventoryPool`
2. In your options dataclass, add `start_inventory_from_pool: StartInventoryPool`

And that's literally everything

Reply:
Oh nice, I thought that was added in by default. Definitely a good one to have 🔥